recipes-connectivity/awsclient/awsclient/awsclient.service

```diff
@@ -5,7 +5,7 @@ After=network.target
 [Service]
 Type=oneshot
 User=root
-ExecStart=/usr/bin/awsclient
+ExecStart=/bin/sh -c '[ $(/usr/bin/jq .awsclient < /mnt/config/aws/config/esec.config | grep start | wc -l) -eq 1 ] && /usr/bin/awsclient || exit 0'
```

Comment:
I think you should remove the `|| exit 0` part, as this only masks any possible errors occurring from awsclient. We want to allow for errors to stop the service.

Reply:
the exit 0 is for the cases

1. file /mnt/config/aws/config/esec.config not exist. the state before onboarding
2. the state is stop, it is the initial state. The customer has not yet accepted the contract.
the awsclient is in waiting state.

Reply:
1. In this case the service should deliberately fail. Either exit with a new error code and echo an error message or simply let the command fail without any further error handling.
2. For this we really should disable the service by default and activate it after a successful onboarding and accepted contract. The check you are introducing with this PR can still be integrated to be sure the contract is not declined at a later point in time.